pkg/syncer/README.md

```diff
@@ -0,0 +1,225 @@
+    NAME: kyverno
+    LAST DEPLOYED: Wed Mar 22 20:43:22 2023
+    NAMESPACE: kyverno
+    STATUS: deployed
+    ...
+    ```
+1. Now Kyverno is running on pcluster
+    ```
+    $ KUBECONFIG=/tmp/kind-pcluster1/kubeconfig.yaml kubectl get pod -n kyverno
+    NAME                      READY   STATUS    RESTARTS   AGE
+    kyverno-9c494576b-dgpjt   1/1     Running   0          78s
+    ```
+1. Create a sample policy in the mailbox workspace to downsync 
+    ```
+    $ kubectl apply -f /tmp/kyverno/sample-policy.yaml
+    policy.kyverno.io/sample-policy created
+    ```
+1. The policy is distributed to pcluster and the generated policy report is upsynced
+  1. On the pcluster
+      ```
+      $ KUBECONFIG=/tmp/kind-pcluster1/kubeconfig.yaml kubectl get policy,policyreport
+      NAME                              BACKGROUND   VALIDATE ACTION   READY
+      policy.kyverno.io/sample-policy   true         enforce           true
+
+      NAME                                            PASS   FAIL   WARN   ERROR   SKIP   AGE
+      policyreport.wgpolicyk8s.io/pol-sample-policy   0      1      0      0       0      56s
+      ```
+  1. On the emailbox workspace
+    ```
+    $ kubectl get policy,policyreport
+    NAME                              BACKGROUND   VALIDATE ACTION   READY
+    policy.kyverno.io/sample-policy   true         enforce           true
+
+    NAME                                            PASS   FAIL   WARN   ERROR   SKIP   AGE
+    policyreport.wgpolicyk8s.io/pol-sample-policy   0      1      0      0       0      77s
+    ```
+
+### See policy reports generated at p-cluster via API Export on workload management workspace.
+1. In the previous case, PolicyReport CRD is deployed as a CRD. In order to share the API across workspaces, we define PolicyReport API as APIBiinding
+1. Go to workload management workspace (`edge`)  
+    ```
+    $ kubectl kcp ws root:edge
+    Current workspace is "root:edge".
+    ```
+1. Create APIResourceSchema and APIExpose for PolicyReport CRD
+    ```
+    $ kubectl apply -f /tmp/kyverno/apischema.policyreports.yaml /tmp/kyverno/apiexport.policyreports.yaml
+    apiresourceschema.apis.kcp.io/v0-0-1.policyreports.wgpolicyk8s.io created
+    apiexport.apis.kcp.io/policy-report created
+    ```
+1. Create APIBindings in the emailbox workspace
+    ```
+    $ kubectl kcp ws root:edge:1lkhy98o1f84q2a3-mb-528a4f03-cb9b-4121-aa57-28c58ed19f22
+    ```
+    ```
+    $ cat << EOL | kubectl apply -f -
+    apiVersion: apis.kcp.io/v1alpha1
+    kind: APIBinding
+    metadata:
+      name: policy-report
+    spec:
+      reference:
+        export:
+          path: root:edge
+          name: policy-report
+    EOL
+    ```
+1. Denature PolicyReport CRD in Kyverno Helm chart
+    ```
+    Name of CRD: policyreports.wgpolicyk8s.io => policyreports.wgpolicyk8s.io.denatured
+    Group in CRD: wgpolicyk8s.io.denatured => wgpolicyk8s.io
```

Comment:
I do not understand exactly what is being said here.  Is this an English representation of some editing activity?

Reply:
My description was poor and wrong. I would express the following replacements in CustomResourceDefinition manifest for policyreport resource definition:
1. Replace `metadata.name: policyreports.wgpolicyk8s.io` with `metadata.name: policyreports.wgpolicyk8s.io.denatured`
2. Replace `spec.group: wgpolicyk8s.io` with `spec.group: wgpolicyk8s.io.denatured`

I will update the README so. Thanks for the review.